.github/workflows/prepare-release-pr.yml

```
@@ -0,0 +1,86 @@
+name: Prepare Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'The type of release to prepare a PR for.'
+        required: true
+        type: choice
+        default: 'stable'
+        options:
+          - stable
+          - alpha
+          - beta
+          - rc
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  prepare_release_pr:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          fetch-depth: 0
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+
+      - name: Set node version to 22
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 22
+          cache: 'pnpm'
+
+      - name: Install deps
+        run: pnpm install
+        env:
+          CYPRESS_INSTALL_BINARY: 0
+
+      - name: Run release
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
```

There is a page that uses the folllowing instead:

```suggestion
          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

- https://github.com/actions/checkout/pull/1184

---

This sounds nice and useful.

> (not tested)

Could you? 🤔

---

I checked this action and it uses the id as well and chatgpt also states, it's the user id.

---

See this commit for an example with the commit id:

- [`feb2f15` (#2981)](https://github.com/faker-js/faker/pull/2981/commits/feb2f15d59558624bdf0de7f5fb9a46001f1dbde)